cmd/builder/internal/builder/main_test.go

```diff
@@ -188,7 +188,7 @@ func TestVersioning(t *testing.T) {
 				cfg.SkipStrictVersioning = true
 				cfg.Exporters = []Module{
 					{
-						GoMod: "go.opentelemetry.io/collector/exporter/otlpexporter v0.97.0",
+						GoMod: "go.opentelemetry.io/collector/exporter/otlpexporter v0.112.0",
```

Comment:
Do we need to bump this and the one above on every release? If so, can we automate it?